src/Command.ts

```diff
@@ -40,7 +57,14 @@ async function fulfillsRequirements (requirements: CommandRequirements, msg: Eri
 export interface CommandRequirements {
 	/** If true, the user must be the bot's owner. */
 	owner?: boolean;
-	/** A list of permission strings the user must have. */
+	/** If true, the message must be sent in a server channel. */
+	guildOnly?: boolean;
```

Comment:
I would absolutely LOVE to see a default client option that would affect all commands but be overridden by the command settings. For example, I try to make all my commands guildOnly as I want to encourage activity in servers. Having to add this to 300 command files is a pain. I only have 1 command that I allow in DMS.

Reply:
Isn't `globalCommandRequirements` already a thing? It should work already if you pass either of these new options as part of that object in the client options.